BUILDING.md

```diff
@@ -13,27 +13,32 @@ For building the manual, see the README in the [Manual repository](https://githu
 Windows
 -------
 
-### Visual studio version ###
+### Visual Studio ###
 
-1.) Download Microsoft Visual Studio 
+1.) Download Microsoft Visual Studio (2022, community edition is free - VS2019 can be used as well).
 
-https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017
+https://visualstudio.microsoft.com/downloads/#
 
-Install "Tools for Visual Studio 2019" and
-select these additional options:
+Make sure "Development Tools for .NET" and "MSBuild" are included in the installtion options.
+
+In the installer tab "Individual components", select these:
 
     - C++ Cmake Tools for Windows
     - MSVC v141 - VS2017 C++ x64/x86 build Tools
 
 2.) Download Qt Creator & Libraries
+Take version 5.9 or minor version above, for instance at time of writing it's 5.15.2.
+
+You can either :
+* use the [official installer](https://doc.qt.io/qt-6/get-and-install-qt.html).
+* or use direct download links from the archive at https://download.qt.io/new_archive/qt, for instance [Qt 5.14.2 here](https://download.qt.io/new_archive/qt/5.14/5.14.2/qt-opensource-windows-x86-5.14.2.exe)
```

Comment:
This a link for the wrong version. I think Qt stopped providing offline installer at some point. The online installer is the only option (for binary releases but building from source is long and complex).